README.md

```diff
@@ -42,13 +43,16 @@ Done! Now you should be able to access the project at **0.0.0.0:8080** or **loca
 
 ## Documentation
 
-- Please follow the rules mention in the [contributor's document](https://github.com/siv2r/kidney-exchange/blob/master/CONTRIBUTING.md) while contributing to this project.
-- [How to create a pull request?](https://github.com/siv2r/kidney-exchange/blob/master/CreatePR.md)
+- Please follow the rules mention in the [contributor's document](docs/CONTRIBUTING.md) while contributing to this project.
+- [How to create a pull request?](docs/CreatePR.md)
 
 ## Get in Touch
 
 Suggestions or Queries ? Please feel free to join the [Discord Group](https://discord.com/invite/d7j63WZV82) !
 
+## License
+Distributed under the GNU General Public License v3.0 License. See [LICENSE](https://github.com/siv2r/kidney-exchange/blob/master/LICENSE) for more information.
```

Comment:
I think you forgot to change this too :)

Reply:
@priyanshuone6 can u check now